compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -926,6 +926,23 @@ static LogicalResult setFftOpConfig(spirv::ResourceLimitsAttr limits,
       workgroupSize);
 }
 
+//===----------------------------------------------------------------------===//
+// Winograd Default Configuration
+//===----------------------------------------------------------------------===//
+
+static LogicalResult setWinogradOpConfig(
+    spirv::ResourceLimitsAttr limits,
+    IREE::LinalgExt::WinogradInputTransformOp op) {
+  // Tiling is already done by tile and decompose, so we only set pipeline and
+  // workgroup size
+  auto pipeline = CodeGenPipeline::SPIRVWinogradVectorize;
+  std::array<int64_t, 3> workgroupSize = {32, 4, 4};
```

Comment:
Are these numbers just placeholders for now? Would be nice to put some comments to explain why them.

Reply:
Yes, that is correct. These are numbers that I used to get good performance on the Navi2 AMD GPU on the sizes in stable diffusion. Sure will add a comment to explain that.